### ISSUE TITLE
Xcode 14 SwiftUI Previews

### ISSUE THREAD

**Issue by jnino-splunk** [open-telemetry/opentelemetry-swift#352]
Hello!

I am trying to use SwiftUI Previews on Xcode 14.1 and am having issues. The Previews fail to load with the following error:

```
HumanReadableSwiftError

SettingsError: noExecutablePath(<IDESwiftPackageStaticLibraryProductBuildable:ObjectIdentifier(0x0000600031dcd590):'OpenTelemetrySdk'>)
```

From reading this thread https://developer.apple.com/forums/thread/707569?page=5 It would seem like the `static` type of various  products in the package file might be the problem. Has anyone else encountered this? Thanks!

P.S. Xcode 13.4.1 seems fine 👍

**Comment by nachoBonafonte** [comment]
@jnino-splunk Is it already fixed with Xcode 14.2 as the linked thread suggests_

**Comment by jnino-splunk** [comment]
Hello, I was just about to close this today actually! Yes 14.2 fixes it. Was able to verify it recently
